docs/sources/whatsnew/whats-new-next/README.md

```diff
@@ -0,0 +1,57 @@
+# Contribute to 'What's New in Grafana Cloud'
+
+To have a feature presented in [What's New in Grafana Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/), add an entry to the [`index.md`](./index.md) file in this directory.
+
+Use the following template, replace any `<VARIABLE>` with the appropriate text, explained later:
+
+```markdown
+## <FEATURE>
+
+<!-- <CONTRIBUTOR> -->
+<!-- <AVAILABILITY> -->
+
+<CLOUD AVAILABILITY> in Grafana <CLOUD OFFERING>
+
+<DESCRIPTION>
+```
+
+## _`CONTRIBUTOR`_
+
+The name of the contributor of the feature.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`AVAILABILITY`_
+
+One of:
+
+- OSS
+- Enterprise
+
+Intended availability of the feature when released outside of Grafana Cloud.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`CLOUD AVAILABILITY`_
+
+One of:
+
+- Generally available
+- Available in public preview
+- Available in private preview
+- Experimental
+
+## _`CLOUD OFFERING`_
+
+One of:
+
+- Cloud Free
+- Cloud Pro
+- Cloud Advanced
+- Cloud
+
+## _`DESCRIPTION`_
+
+Include an overview of the feature and problem it solves, and where to learn more.
+Link to any appropriate documentation.
+You must use relative path references when linking to documentation within the Grafana repository.
+Use the Hugo `relref` shortcode for build time link checking.
+For more information about the `relref` shortcode, refer to [Links and references](https://grafana.com/docs/writers-toolkit/writing-guide/references/).
```

Comment:
````suggestion
For more information about the `relref` shortcode, refer to [Links and references](https://grafana.com/docs/writers-toolkit/writing-guide/references/).

## Example

```
**Updated navigation**

<!-- Jack Baldry -->
<!-- OSS, Enterprise -->

_Available in public preview in Grafana Cloud Pro and Advanced_

The navigation in Grafana Cloud has been updated in the following ways...

```
````

Reply:
The example differs from the template in the following ways:

1. Uses bold emphasis instead of an H2 heading.
1. Adds italic emphasis _`CLOUD AVAILABILITY`_
1. Adds "Available" prefix to _`CLOUD AVAILABILITY`_
1. Specifies multiple cloud offerings when the template says "One of"

I would propose that I should retrofit your example into the template with the exception of the bold emphasis which I should keep as an H2 heading. WDYT?

Reply:
1. I only used bold instead of an H2 because I wasn't sure if it would mess things up, but I'm assuming if it's in the code block, it'll be okay?
2. I'm copying what we do in the OSS What's new.
3. I'm copying what we do in the OSS What's new.
4. I didn't catch that you were treating the Cloud offerings list like the OSS, Enterprise thing. I don't think we should force users to parse that Free means Free+, or Pro means Pro+. Just list them out like we do in the OSS What's new. I'd suggest we change the template.